logstash-output-elasticsearch.gemspec

```diff
@@ -25,6 +25,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'ftw', '~> 0.0.42'
   s.add_development_dependency 'addressable', "~> 2.3.0" # used by FTW. V 2.5.0 is ruby 2.0 only.
   s.add_development_dependency 'logstash-codec-plain'
+  s.add_development_dependency 'json' # used by spec/unit/outputs/elasticsearch/http_client/pool_spec.rb
```

Comment:
I think you're right, I think it is a development dependency.
Logstash core does reference the JSON gem, and we only `LogStash::Util::Json` in the runtime portion of the code, but I guess the tests use JSON directly.